NOLF2/ClientShellDLL/ClientShellShared/ProfileMgr.cpp

```diff
@@ -1066,7 +1067,8 @@ void CUserProfile::ImplementMusicVolume()
 
 void CUserProfile::ApplyDisplay()
 {
-
+	SDL_Window* window;
```

Comment:
Again this can be moved to line 1120, window is not used anywhere else in the method